src/main/java/org/quartzpowered/engine/object/Component.java

```diff
@@ -36,7 +36,8 @@
 public abstract class Component {
     private static final Field objectField = ReflectionUtil.getConstantField(Component.class, "gameObject");
 
-    @Inject private Logger logger;
+    @Inject
+    private Logger logger;
 
```

Comment:
These are supposed to be on one line, please allow one line annotations in your formatting options and update the PR.